presentation-compiler/src/test/scala/org/alephium/ralph/lsp/pc/search/gotodef/GoToLocalVariableSpec.scala

```diff
@@ -0,0 +1,76 @@
+package org.alephium.ralph.lsp.pc.search.gotodef
+
+import org.alephium.ralph.lsp.pc.search.TestCodeProvider._
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class GoToLocalVariableSpec extends AnyWordSpec with Matchers {
+
+  "return empty" when {
+    "variable does not exist" in {
+      goTo(
+        """
+          |Contract GoToTest() {
+          |
+          |  pub fn function() -> () {
+          |    // varB does not exists
+          |    let varA = v@@arB
+          |  }
+          |
+          |}
+          |""".stripMargin
+      )
+    }
+  }
+
+  "return non-empty" when {
+    "single local variable exists" in {
+      goTo(
+        """
+          |Contract GoToTest() {
+          |
+          |  pub fn function() -> () {
+          |    >>let varA = 123
+          |    <<let varB = var@@A
+          |  }
```

Comment:
aaaah shit, this is because the indexes aren't properly set in the node, I know exactly the problem. I'll try to fix that in the node soon.